website/pages/en/404.js

```diff
@@ -0,0 +1,36 @@
+const React = require("react");
+
+
+class ErrorPage extends React.Component {
+    render() {
+        return (
+
+            <div className="error-page">
+                <div className="container">
+                    <div className="row">
+                        <div className="col-md-12 error-message">
```

Comment:
Some of these styles won't mean anything anymore like `col-md-12` because we don't have bootstrap anymore

Reply:
I'll remove them, are we using another framework?

Reply:
It's just whatever docusaurus provides and our own styles - The black heading was what we had before but I don't think we need that

Reply:
roger that, I want to work in something eye-catching as you mentioned before. I removed the bootstrap classes